scanners/android/android_test.go

```diff
@@ -1,38 +1,46 @@
 package android
 
 import (
-	"fmt"
-	"path/filepath"
 	"testing"
 
-	"github.com/bitrise-io/bitrise-init/models"
-	"github.com/bitrise-io/go-utils/command/git"
 	"github.com/stretchr/testify/require"
 )
 
-func gitClone(t *testing.T, dir, uri string) {
-	fmt.Printf("cloning into: %s\n", dir)
-	g, err := git.New(dir)
-	require.NoError(t, err)
-	require.NoError(t, g.Clone(uri).Run())
-}
-
-func Test_detect(t *testing.T) {
```

Comment:
This test is removed because there is an integration test scanning the same repository.